firewall.conf

```diff
@@ -15,9 +15,9 @@ iptables -A INPUT -p tcp --dport ssh -j ACCEPT
 # Allow for all traffic from anywhere for the following services
 iptables -A INPUT -p tcp --dport 80 -j ACCEPT          # nginx
 iptables -A INPUT -p tcp --dport 443 -j ACCEPT         # nginx (SSL)
-iptables -A INPUT -p tcp --dport 2812 -j ACCEPT        # monit dashboard
+#iptables -A INPUT -p tcp --dport 2812 -j ACCEPT        # monit dashboard
```

Comment:
Disabling those services requires AppDashboard modifications as well.
We'll need to disable the respective sidebar tabs, for example.